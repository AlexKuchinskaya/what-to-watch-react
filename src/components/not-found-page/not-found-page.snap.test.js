import React from 'react';
import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';
import NotFoundPage from './not-found-page';

test(`Should NotFoundScreen render correctly`, () => {
  const history = createMemoryHistory();
  const {container} = render(
      <Router history={history}>
        <NotFoundPage />
      </Router>
  );
  expect(container).toMatchSnapshot();
});
