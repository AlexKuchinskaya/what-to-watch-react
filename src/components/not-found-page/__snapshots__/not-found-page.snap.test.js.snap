// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should NotFoundScreen render correctly 1`] = `
<div>
  <section
    class="movie-card"
  >
    <header
      class="page-header movie-card__head"
    >
      <div
        class="logo"
      >
        <a
          class="logo__link "
          href="/"
        >
          <span
            class="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            class="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            class="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
    </header>
    <section
      class="game__screen"
    >
      <h1>
        404. Page not found
      </h1>
      <a
        href="/"
      >
        Вернуться на главную
      </a>
    </section>
  </section>
</div>
`;
